godot-codegen/src/tests.rs

```diff
@@ -160,3 +162,55 @@ fn test_enumerator_names() {
         */
     ];
 }
+
+#[test]
+fn test_parse_native_structures_format() {
+    // Convenience constructor.
+    fn native(ty: &str, name: &str) -> NativeStructuresField {
+        NativeStructuresField {
+            field_type: String::from(ty),
+            field_name: String::from(name),
+        }
+    }
+
+    assert_eq!(parse_native_structures_format("").unwrap(), vec![],);
+
+    assert_eq!(
+        parse_native_structures_format("float a;Vector3 zzz; real_t zzz;    Vector2 foobar ")
+            .unwrap(),
+        vec![
+            native("float", "a"),
+            native("Vector3", "zzz"),
+            native("real_t", "zzz"),
+            native("Vector2", "foobar"),
+        ],
+    );
```

Comment:
Do we need to support these extra whitespaces before/after a type-name pair?
If not, I'd avoid making the spec more complex.

Reply:
Good point. It doesn't look like the current extension API has any excess whitespace. Removed irrelevant test.